cmake/modules/BuildPlugin.cmake

```diff
@@ -9,7 +9,7 @@ MACRO(BUILD_PLUGIN PLUGIN_NAME)
 	CMAKE_PARSE_ARGUMENTS(PLUGIN "" "" "MOCFILES;EMBEDDED_RESOURCES;UICFILES;LINK" ${ARGN})
 	SET(PLUGIN_SOURCES ${PLUGIN_UNPARSED_ARGUMENTS})
 
-	INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR} ${CMAKE_BINARY_DIR} ${CMAKE_SOURCE_DIR}/include ${CMAKE_SOURCE_DIR}/src/gui)
+	INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR} ${CMAKE_BINARY_DIR} ${CMAKE_SOURCE_DIR}/include)
```

Comment:
Update, this appears to [date back 10 years](https://github.com/gnudles/lmms/blob/e637495/cmake/modules/BuildPlugin.cmake#L50) and I see no evidence it's needed as headers aren't in that directory.  Should be safe to remove. 👍